src/thir.md

```diff
@@ -19,8 +20,9 @@ the types have been filled in, which is possible after type checking has complet
 But it has some other interesting features that distinguish it from the HIR:
 
 - Like the MIR, the THIR only represents bodies, i.e. "executable code"; this includes
-  function bodies, but also `const` initializers, for example. Consequently, the THIR
-  has no representation for items like `struct`s or `trait`s.
+  function bodies, but also `const` initializers, for example. Specifically, all [body owners] have
+  THIR created. Consequently, the THIR has no representation for items like `struct`s or `trait`s.
+  References to those items are represented as `HirId`s in the THIR.
```

Comment:
This isn't correct. `HirId`s only exist for the local crate, but we need to be able to reference items from other crates as well. I'm not sure this sentence is really needed though.
```suggestion
```